packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -57,7 +57,8 @@ export const sendBoxStyle = mergeStyles({
   },
   ':-moz-placeholder': {
     fontSize: '0.875rem'
-  }
+  },
+  padding: '0 2rem 0 0.5rem'
```

Comment:
note: this will need a condition for RTL, I think the fix for this should be in the send box component to use a stackbox, something like:
```
<Stack horizontal>
  <Stack.Item grow>{textArea}</StackItem>
  <Stack.Item>{sendIcon}</Stack.Item>
</Stack> 
```
(^that code is speculative, haven't tried but semantically it should be something like that)

Reply:
Turns out RTL flips padding, that's fun!
![image](https://user-images.githubusercontent.com/2684369/126842608-f7a938d5-1730-47c8-b169-81cdfadde84c.png)
![image](https://user-images.githubusercontent.com/2684369/126842611-d22f3f24-d5c8-4da7-ae09-405e1f0f781b.png)

Reply:
And doing stacks won't work because the icon needs to be floating inside the textbox to have it encased inside the textbox (and fluentUI doesn't provide clickable icons within text boxes)